### PR TITLE
Add Contributor Code of Conduct

### PR DESCRIPTION
**Pull request by jameslamb**

### CODE REVIEW

File: CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+## Our Standards
+
+Examples of behavior that contributes to creating a positive environment include:
+
+Using welcoming and inclusive language
+Being respectful of differing viewpoints and experiences
+Gracefully accepting constructive criticism
+Focusing on what is best for the community
+Showing empathy towards other community members
+Examples of unacceptable behavior by participants include:
```
Comment (austin3dickey) [inline CONDUCT.md]: Can you add a line break before this line, and possibly make these lists bulleted?

File: CONDUCT.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+The use of sexualized language or imagery and unwelcome sexual attention or advances
+Trolling, insulting/derogatory comments, and personal or political attacks
+Public or private harassment
+Publishing others' private information, such as a physical or electronic address, without explicit permission
+Other conduct which could reasonably be considered inappropriate in a professional setting
+Our Responsibilities
```
Comment (austin3dickey) [inline CONDUCT.md]: Line break here too

File: CONDUCT.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+## Scope
+
+This Code of Conduct applies both within project spaces and in public spaces when an individual is representing the project or its community. Examples of representing a project or community include using an official project e-mail address, posting via an official social media account, or acting as an appointed representative at an online or offline event. Representation of a project may be further defined and clarified by project maintainers.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be reported by contacting the project team at [INSERT EMAIL ADDRESS]. All complaints will be reviewed and investigated and will result in a response that is deemed necessary and appropriate to the circumstances. The project team is obligated to maintain confidentiality with regard to the reporter of an incident. Further details of specific enforcement policies may be posted separately.
```
Comment (austin3dickey) [inline CONDUCT.md]: haha gotta insert email address

Comment (jameslamb) [inline CONDUCT.md]: lol I went too fast, good catches

**Pull request merged**